document/core/util/mathjax2katex.py

```diff
@@ -225,22 +225,24 @@ def ExtractMath(match):
   sys.stderr.write('Processing %d fragments.\n' % len(fixups))
 
   done_fixups = []
+  success = True
 
   def Worker():
+    nonlocal success
     while True:
       cls_before, cls_after, spans, mth, start, end = q.get()
       try:
         fixed = ('class="' + cls_before + ' ' + cls_after + '">' +
                  spans + ReplaceMath(cache, mth) + '<')
         done_fixups.append((start, end, fixed))
       except Exception:
-        sys.stderr.write('!!! Error processing fragment')
+        success = False
 
       q.task_done()
       sys.stderr.write('.')
 
   q = queue.Queue()
-  for i in range(40):
+  for i in range(len(os.sched_getaffinity(0))):
```

Comment:
How does this change the amount of time to process all the fragments (without caching) for you?

Reply:
I can do a test, but I would say not noticeable. It's already pretty slow anyway :)

Reply:
Here's results, I put a `time` before `python util/mathjax2katex.py ...`:

## cached 

| | 40 | os.sched_getaffinity(0) |
| - | -- | --- |
|real |   0m9.893s  |    0m10.051s |
| user|   4m3.413s |    4m1.608s|
| sys|    0m17.806s |     0m17.543s|

## clean build (remove cache)

|  | 40 | os.sched_getaffinity(0) |
| - | -- | --- |
| real|    1m17.899s |    1m27.676s |
| user|    22m23.613s |     21m52.605s |
|sys |     1m33.117s |     1m29.121s |

So yea not that much on my 28 core cpu.
But try 40 threads on a single/duo core :)